client/lib/components/learn_page_promo.dart

```diff
@@ -6,6 +8,22 @@ import 'package:who_app/components/promo_curved_background.dart';
 import 'package:who_app/components/themed_text.dart';
 import 'package:who_app/constants.dart';
 
+const double blurRadius = 20;
+const List<Shadow> headerTextShadows = [
```

Comment:
Can we make it less verbose? Something like:
```suggestion
const shadow = Shadow(blurRadius: blurRadius);
// 4X shadows is used to make it more opaque and offset from the background
const opaqueShadow = <Shadow>[
   shadow,
   shadow,
   shadow,
   shadow]
```
The reason to use `opaqueShadow` is that it more closely matches the function. You could imagine using this on other text.... with a constant that's shared more widely on other pages.

See Dart type system for preference in specifying type on the `[]`:
https://dart.dev/guides/language/type-system